docs/DeveloperGuide.md

```diff
@@ -249,29 +249,42 @@ _{Explain here how the data archiving feature will be implemented}_
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
+
 | Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
 | -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| `* * *`  | user                                       | View my tuition applications   | I can keep track of the tuitions I that I have applied                 |
+| `* * *`  | user                                       | Delete a tuition application   | I can reverse the tuition application                                  |
```

Comment:
(minor) Will removing 'I can' be better?

Reply:
Yup! i already changed it when i merged for kingsley's part so @jared98lyj could u resolve the merge conflicts, just delete my part and take kingsley's portion for this because his is complete.

Reply:
Yup! i will change it.